.pre-commit-config.yaml

```diff
@@ -1,3 +1,5 @@
+default_language_version:
```

Comment:
Why is this needed?

Reply:
I added that due to an error I was getting uwing the `pre-commit` with conda environment. I followed the suggestion from this post [here](https://stackoverflow.com/questions/72177222/runtimeerror-failed-to-find-interpreter-for-builtin-discover-of-python-spec-py) and it worked in my case.